tests/utils_tests.py

```diff
@@ -237,3 +242,30 @@ def makeExposure(flipX=False, flipY=False):
     exposure.setFilter(afwImage.FilterLabel(band='g'))
 
     return exposure
+
+
+def makeRegionTime(exposure=None, begin=None, end=None):
+    """Make a `RegionTimeInfo` for testing
+
+    Parameters
+    ----------
+    exposure : `lsst.afw.image.Exposure`, optional
+        Exposure to construct a ``RegionTimeInfo`` for.
+        If None, a default exposure will be created.
+
+    Returns
+    -------
+    `lsst.pipe.base.utils.RegionTimeInfo`
+        Object containing the spatial region and temporal timespan for an exposure.
```

Comment:
It probably won't matter since these docs won't be built, but [please name your return value](https://developer.lsst.io/python/numpydoc.html#py-docstring-returns) so that it's Sphinx-compatible.